subworkflows/nf-core/fastq_fastqc_umitools_trimgalore/main.nf

```diff
@@ -36,28 +36,28 @@ workflow FASTQ_FASTQC_UMITOOLS_TRIMGALORE {
     fastqc_html = Channel.empty()
     fastqc_zip  = Channel.empty()
     if (!skip_fastqc) {
-        FASTQC (reads)
+        FASTQC (ch_reads)
         fastqc_html = FASTQC.out.html
         fastqc_zip  = FASTQC.out.zip
         ch_versions = ch_versions.mix(FASTQC.out.versions.first())
     }
 
-    umi_reads = reads
+    umi_reads = ch_reads
     umi_log   = Channel.empty()
     if (with_umi && !skip_umi_extract) {
-            UMITOOLS_EXTRACT (reads)
-            umi_reads   = UMITOOLS_EXTRACT.out.reads
+            UMITOOLS_EXTRACT (ch_reads)
+            umi_reads   = UMITOOLS_EXTRACT.out.ch_reads
             umi_log     = UMITOOLS_EXTRACT.out.log
             ch_versions = ch_versions.mix(UMITOOLS_EXTRACT.out.versions.first())
 
             // Discard R1 / R2 if required
             if (umi_discard_read in [1,2]) {
                 UMITOOLS_EXTRACT
                     .out
-                    .reads
+                    .ch_reads
                     .map {
-                        meta, reads ->
-                            meta.single_end ? [ meta, reads ] : [ meta + ['single_end': true], reads[umi_discard_read % 2] ]
+                        meta, ch_reads ->
```

Comment:
I don't think you should change it here to ch_reads.
As you are inside a closure, I think this should be reads.